public/js/icinga/ui.js

```diff
@@ -591,12 +591,12 @@
                         position: 'fixed'
                     });
                 }
-            });
+            });*/
 
             this.fixControls($container);
         },
 
-        fixControls: function($container) {
+        fixControls2: function($container) {
```

Comment:
What's up with the fixControls2?
It's not being used anywhere -> could it just be hidden behind a comment if it is unused?